src/persist/benches/writer.rs

```diff
@@ -27,29 +61,59 @@ fn bench_write_sync<L: Log>(writer: &mut L, data: Vec<u8>, b: &mut Bencher) {
     })
 }
 
+// Benchmark the write throughput of Blob::set.
+fn bench_set<B: Blob>(writer: &mut B, data: Vec<u8>, b: &mut Bencher) {
+    // We need to pick a random starting key because Criterion likes to run this
+    // function many times as part of a warmup, and if we deterministically start
+    // with the same starting key, we will overwrite it.
+    let mut key = rand::random::<usize>();
+    b.iter(|| {
+        futures_executor::block_on(writer.set(&format!("{}", key), data.clone(), false))
```

Comment:
is it important that the keys written by the benchmark be sequential? if so, let's document why. if not, we could just always write to a random key

here and elsewhere